### PR TITLE
Fix Issue#856:Toggle status text (Online/offline)

### LINKED ISSUE #856: Toggle text on mode change
**Summary:** 

change the text on mode change.

**Steps to reproduce:** 

1)run the app
2)toggle from offline to online

**Expected behavior:** 

No text change

**Observed behavior:** 

Text change from offline to online and vice versa

**Gif**

I am working on this

### PR DESCRIPTION
**Pull request by AkshGautam**
Fix #856 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
@therajanmaurya @tarun0  Please review fixes for #856

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java [hunk 1]
```diff
@@ -233,7 +233,10 @@ public boolean onNavigationItemSelected(MenuItem item) {
     public void setupUserStatusToggle() {
         userStatusToggle
                 = (SwitchCompat) mNavigationHeader.findViewById(R.id.user_status_toggle);
+        final TextView user_status = ButterKnife.
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java]: This method is deprecated in the newer version and Android's `findViewById` can do the same thing https://stackoverflow.com/questions/45516753/butterknife-findbyid-method-is-deprecated-in-version-8-8-0

Also, please follow camelCase for variable-naming

Comment (AkshGautam) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java]: sure, will resolve this

**Comment by droidchef** [comment]
Requested changes never made it back to the PR.

**Pull request closed without merging**